content/shmem_query_interoperability.tex

```diff
@@ -0,0 +1,47 @@
+\apisummary{
+    Determines whether an interoperability feature is supported by the \openshmem
+    library implementation.
+}
+\begin{apidefinition}
+
+\begin{Csynopsis}
+int @\FuncDecl{shmem\_query\_interoperability}@(int property);
+\end{Csynopsis}
+
+\begin{apiarguments}
+    \apiargument{IN}{property}{The interoperability property queried by the user.}
+\end{apiarguments}
+
+% compiling error ?
+% \apidescription{
```

Comment:
Do we need this?

Reply:
I fixed the compiling issue by adding `\apinotes{None.}` after `\apidescription{...}`. It is now consistent with other API definition.